.dockerignore

```diff
@@ -0,0 +1,2 @@
+Dockerfile
```

Comment:
None of these files need to exist inside the images.

Feel free to add more things, I didn't want to go crazy since I dunno what is going on. It's pretty common to add `.git` so you don't pull all of git history into the image, etc.

This only affects the `COPY . /usr/src/zeus` line.